controller/array_action/array_mediator_svc.py

```diff
@@ -215,25 +220,27 @@ def _generate_snapshot_response_with_verification(self, cli_object):
             raise controller_errors.ExpectedSnapshotButFoundVolumeError(cli_object.name, self.endpoint)
         return self._generate_snapshot_response(cli_object, fcmap.source_vdisk_name)
 
-    def _get_cli_volume(self, volume_name_or_id, not_exist_err=True):
+    def _get_cli_volume(self, volume_name, not_exist_err=True):
```

Comment:
why its so different then `_get_cli_volume_by_wwn`? can't we unite them with some `is_id` flag for `filter_value`?

Reply:
they handle different errors, so most of their code is different (not just the `filter_value`).
I would leave them separated.